ubuntu-github-actions-openvino--20.04/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# Version: 20230724
+# Image name: quay.io/opencv-ci/opencv-ubuntu-20.04-openvino
+
+FROM openvino/ubuntu20_dev
```

Comment:
Should we apply a specific tag? `FROM openvino/ubuntu20_dev:2023.0.0`?

Reply:
> Should we apply a specific tag?

I agree, we should use version tag.

Reply:
Agree. Done!